content/applications/services/helpdesk/overview/sla.rst

```diff
@@ -2,46 +2,175 @@
+   :abbr:`SLA (Service Level Agreement)` was titled `2 Days to Start`, it tracks the working time
+   before work on a ticket has begun, and would have `In Progress` as the :guilabel:`Reach Stage`.
+
+Meeting SLA deadlines
+=====================
+
+Once it is determined that a ticket fits the criteria of an :abbr:`SLA (Service Level Agreement)`
+policy, a deadline is calculated. The deadline is based on the creation date of the ticket, and the
+targeted working hours. The deadline is then added to the ticket, as well as a white tag indicating
+the name of the :abbr:`SLA (Service Level Agreement)` applied.
+
+.. image:: sla/sla-open-deadline.png
+   :align: center
+   :alt: View of a ticket's form emphasizing an open SLA deadline on a ticket in Odoo Helpdesk
+
+.. important::
+   If a ticket fits the criteria for more than one :abbr:`SLA (Service Level Agreement)`, the
+   earliest occurring deadline will be displayed on the ticket. Once that deadline has passed, the
+   next deadline will be displayed.
+
+Once a ticket satisfies an :abbr:`SLA (Service Level Agreement)` policy, the :abbr:`SLA (Service
+Level Agreement)` tag turns green, and the :guilabel:`Deadline` field disappears.
+
+.. image:: sla/sla-deadline.png
    :align: center
-   :height: 330
-   :alt: View of a ticket’s form emphasizing a satisfied SLA in Odoo Helpdesk
+   :alt: View of a ticket's form emphasizing a satisfied SLA in Odoo Helpdesk
+
+If the :abbr:`SLA (Service Level Agreement)` deadline passes and the ticket has not moved to the
+:guilabel:`Reach Stage`, the :abbr:`SLA (Service Level Agreement)` tag will turn red. Once the
+:abbr:`SLA (Service Level Agreement)` has failed, the red tag will stay on the ticket, even after
+the ticket is moved to the :guilabel:`Reach Stage`.
+
+.. image:: sla/sla-passing-failing.png
+   :align: center
+   :alt: View of a ticket's form with a failing and passing SLA in Odoo Helpdesk
+
+Analyzing SLA performance
+=========================
+
+The :guilabel:`SLA Status Analysis` report tracks how quickly an :abbr:`SLA (Service Level
+Agreement)` is fulfilled, as well as the success rate of individual policies. It is found under
+:menuselection:`Helpdesk --> Reporting --> SLA Status Analysis`.
+
+Using the Pivot view
+--------------------
+
+By default, the report is displayed in a :guilabel:`Pivot` view, and is filtered to show the number
+of SLAs failed and the failure rate over the last 30 days, grouped by team.
+
+.. image:: sla/sla-status-analysis.png
+   :align: center
+   :alt: View of the SLA status analysis report in Odoo Helpdesk
+
+To add the number of SLAs passed or in progress, click :guilabel:`Measures` and choose from the
+options available.
+
+To add a group to a row or column, click the plus button (:guilabel:`➕`) next to :guilabel:`Total`,
+and then select one of the groups. To remove one, click the minus button (:guilabel:`➖`) and
+deselect.
```

Comment:
I don't have a problem with this change, but I took this setup from a currently published doc (https://www.odoo.com/documentation/16.0/applications/general/reporting.html#using-the-pivot-view) to keep consistent, just in case we want to test this doc with other devices